mdlinkcheckconfig.json

```diff
@@ -24,6 +24,9 @@
     {
       "pattern": "https://doi.org/10.7910/DVN/TJCLKP"
     },
+    {
```

Comment:
When running on CI, both of these links were failing so I added them to the ignore list. ospoplusplus serves just a blank page these days, I'm not sure if it has shut down or what.

Reply:
It has shut down.